pydoctor/themes/base/function-child.html

```diff
@@ -8,6 +8,7 @@
     <t:attr name="name" t:render="shortFunctionAnchor" />
```

Comment:
When a template changed, we track it into a independent version system such that customizations that overrides this template will get a warning saying they should update.

Can you increase the `<meta name="pydoctor-template-version" content="2" />` to `<meta name="pydoctor-template-version" content="3" />` please?